README.md

```diff
@@ -1,28 +1,52 @@
 <img src="https://www.kubeflow.org/images/logo.svg" width="100">
 
-[![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/2660/badge)](https://bestpractices.coreinfrastructure.org/projects/2660) [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/kubeflow/kubeflow/badge)](https://securityscorecards.dev/viewer/?uri=github.com/kubeflow/kubeflow) [![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/kubeflow/badge)](https://clomonitor.io/projects/cncf/kubeflow)
+[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels)
+[![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/kubeflow/badge)](https://clomonitor.io/projects/cncf/kubeflow)
 
-Kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
+The Kubeflow makes AI/ML on Kubernetes simple, portable and scalable.
 
 ---
 
 ## Documentation
+
 Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
 
-## Working Groups
-The Kubeflow community is organized into working groups (WGs) with associated repositories, that focus on specific pieces of the ML platform. 
+## Kubeflow Components
+
+The Kubeflow ecosystem is composed of multiple open-source projects for each stage in
+[the ML lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle).
+
+Learn more about each project in [the Kubeflow documentation](https://www.kubeflow.org/docs/components/).
+
+Please use the following GitHub repositories to open issues and pull requests for
+[the different Kubeflow components](https://www.kubeflow.org/docs/started/introduction/#what-are-standalone-kubeflow-components):
+
+| Component                  | Source Code                                                                   |
```

Comment:
Should dashboard be added to this list? The repo was just created.

Reply:
@james-jwu I added it below, please check: https://github.com/kubeflow/kubeflow/blob/ddbe0ec5b4c42864374cdf2f04164e55b60d2a42/README.md#:~:text=Use%20the%20kubeflow/dashboard%20GitHub%20repository%20for%20the%20Kubeflow%20Profile%20Controller%2C%20Central%20Dashboard%2C%20CRUD%20Web%20Apps%2C%20PVC%20Viewer%2C%20PodDefault%2C%20and%20Access%20Management%20components.